isovar/variant_read.py

```diff
@@ -148,8 +150,13 @@ def variant_read_from_single_read_at_locus(read, ref, alt):
                     read))
             return None
 
+    if sequence[read_pos_before + 1:read_pos_after] != alt:
```

Comment:
I think that R1 does not agree with delGA since it's up to the variant caller to call the largest possible variant there. I would think of delGAGA as a totally distinct variant.